crates/astria-conductor/src/executor/mod.rs

```diff
@@ -161,12 +166,30 @@ impl Executor {
                             let block_subset =
                                 SequencerBlockSubset::from_sequencer_block_data(*block, &self.chain_id);
 
-                            if let Err(e) = self.execute_block(block_subset).await {
-                                error!(
-                                    sequencer_block_height = height,
-                                    error = ?e,
-                                    "failed to execute block"
-                                );
+                            let executed_block_result = self.execute_block(block_subset).await;
+                            match executed_block_result {
+                                Ok(Some(executed_block)) => {
+                                    if let Err(e) = self.update_soft_commitment(executed_block.clone()).await {
+                                        error!(
+                                            height = height,
+                                            error = ?e,
+                                            "failed to update soft commitment"
+                                        );
+                                    }
+                                }
+                                Ok(None) => {
+                                    error!(
+                                        height = height,
+                                        "block execution with empty return"
+                                    );
+                                }
```

Comment:
this isn't an error, if empty block execution is disabled this is fine (and there's already a debug log for that case) so maybe just ignore this case?

Reply:
you are correct! i'll remove the error! macro